inc/saiuburst.h

```diff
@@ -380,23 +380,57 @@ typedef sai_status_t (*sai_get_tam_histogram_attribute_fn)(
  * @count counter_ids[number_of_counters]
  * @count counters[number_of_counters]
  */
-typedef sai_status_t (*sai_get_tam_histogram_stats_fn)(
-        _In_ sai_object_id_t tam_histogram_id,
+typedef sai_status_t (*sai_get_tam_microburst_stats_fn)(
+        _In_ sai_object_id_t tam_microburst_id,
+        _In_ uint32_t number_of_counters,
+        _In_ const sai_stat_id_t *counter_ids,
+        _Out_ uint64_t *counters);
+
+/**
+ * @brief Obtain the values for all bins from a histogram.
+ *
+ * @param[in] tam_microburst_id Microburst id
+ * @param[in] number_of_counters Number of counters in the array
+ * @param[in] counter_ids Specifies the array of counter ids
+ * @param[in] mode Statistics mode
+ * @param[out] counters Array of resulting counter values.
+ *
+ * @return #SAI_STATUS_SUCCESS on success, failure status code on error
+ */
+typedef sai_status_t (*sai_get_tam_microburst_stats_ext_fn)(
+        _In_ sai_object_id_t tam_microburst_id,
         _In_ uint32_t number_of_counters,
         _In_ const sai_stat_id_t *counter_ids,
+        _In_ sai_stats_mode_t mode,
         _Out_ uint64_t *counters);
 
+/**
+ * @brief Clear port pool statistics counters.
```

Comment:
Typo in brief. I guess it should clear TAM microburst stats

Reply:
fixed